mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -194,75 +181,21 @@ public void run() {
-                    @Override
-                    public void onError(Throwable e) {
-
-                    }
-
-                    @Override
-                    public void onNext(Client client) {
-                        // add name to profile
-                        String name = client.getDisplayName();
-                        TextView textViewUsername = (TextView) findViewById(R.id.tv_user_name);
-                        textViewUsername.setText(name);
-
-                        // download image
-                        if (client.isImagePresent()) {
-                            String url = PrefManager.getInstanceUrl()
-                                    + "/"
-                                    + "clients/"
-                                    + userId
-                                    + "/images?maxHeight=120&maxWidth=120";
-                            GlideUrl glideUrl = new GlideUrl(url, new LazyHeaders.Builder()
-                                    .addHeader(MifosInterceptor.HEADER_TENANT, PrefManager
-                                            .getTenant())
-                                    .addHeader(MifosInterceptor.HEADER_AUTH, PrefManager
-                                            .getToken())
-                                    .addHeader("Accept", "application/octet-stream")
-                                    .build());
-
-                            ImageView imageViewUserPicture = (ImageView) findViewById(R.id
-                                    .iv_user_picture);
-                            Glide.with(getApplicationContext())
-                                    .load(glideUrl)
-                                    .asBitmap()
-                                    .error(R.drawable.ic_account_circle)
-                                    .placeholder(R.drawable.ic_account_circle)
-                                    .into(new BitmapImageViewTarget(imageViewUserPicture) {
-                                        @Override
-                                        protected void setResource(Bitmap result) {
-                                            // check a valid bitmap is downloaded
-                                            if (result == null || result.getWidth() == 0)
-                                                return;
-
-                                            // set to image view
-                                            ImageView imageViewUserPicture = (ImageView)
-                                                    findViewById(R.id.iv_user_picture);
-                                            imageViewUserPicture.setImageBitmap(result);
-                                        }
-                                    });
-                        }
-                    }
-                });
+        // download logged in user
+        final User loggedInUser = PrefManager.getUser();
+
+        TextView textViewUsername = (TextView) mNavigationHeader.findViewById(R.id.tv_user_name);
```

Comment:
Since we are using ButterKnife please avoid the use of findViewById. This ensures that the code structure is same throughout the app.